File: matlab/set_params.m
params.H = 160;       % loaded video height size
params.W = 240;       % loaded video width size

params.patchW = 40;   % 3D patch spatial width
params.patchH = 40;   % 3D patch spatial height
params.tprLen = 16;   % 3D patch temporal length

params.BKH = params.H / params.patchH;      % region number in height
params.BKW = params.W / params.patchW;      % region number in width

params.srs = 40;       % spatial sampling rate in video volume
params.trs = 16;       % temporal sampling rate in video volume

params.MT_thr = 5;    % 3D patch selecting threshold

File: set_params.py
H = 160                   # loaded video height size
W = 240                   # loaded video width size

patchH = 40               # height of 3D patch
patchW = 40               # width of 3D patch
tprLen = 16               # temporal length of 3D patch

BKH = H / patchH          # region number in height
BKW = W / patchW          # region number in width

srs = 40                  # spatial sampling rate in training video volume
trs = 16                  # temporal sampling rate in training video volume

PCAdim = 32               # PCA Compression dimension

MT_thr = 5                # 3D patch selecting threshold

k = 128                   # number of atoms in the dictionary, if k is None, select k = round(0.2 * n_samples)
